runelite-client/src/main/java/net/runelite/client/plugins/cannon/CannonConfig.java

```diff
@@ -25,28 +25,43 @@
 package net.runelite.client.plugins.cannon;
 
 import java.awt.Color;
-import net.runelite.client.config.Alpha;
-import net.runelite.client.config.Config;
-import net.runelite.client.config.ConfigGroup;
-import net.runelite.client.config.ConfigItem;
+
+import net.runelite.client.config.*;
 
 @ConfigGroup("cannon")
 public interface CannonConfig extends Config
 {
 	@ConfigItem(
-		keyName = "showEmptyCannonNotification",
-		name = "Empty cannon notification",
-		description = "Configures whether to notify you that the cannon is empty"
+		keyName = "enableCannonNotifications",
+		name = "Enable cannon notifications",
+		description = "Configures whether to notify you when your cannon has X cannonballs remaining.",
+		position = 1
```

Comment:
Why does it need position

Reply:
Without a position it sorts of alphabetically. Generally we try to group related options next to each other though.